.github/workflows/tests.yaml

```diff
@@ -54,6 +54,12 @@ jobs:
         shell: bash -l {0}
         run: conda config --show
 
+      - name: Optionally install pickle5
+        shell: bash -l {0}
+        # pickle5 is used for out-of-band pickling support in Python 3.7
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.python-version < '3.8' }}
+        run: mamba install -c conda-forge pickle5
```

Comment:
It might be worth limiting this to only some Python 3.7 CI jobs given it is still useful to test without `pickle5`. Though it would be good to have some Python 3.7 jobs that include this.

Not sure how we want to handle that (see `partition`s above, which might be a useful way to handle this). Though maybe there are better ways.

Reply:
Addressed by testing both pickle protocol 4 & 5 in PR ( https://github.com/dask/distributed/pull/5313 ).